src/pages/components/input-group.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Input Group
+description: >
+  An Input Group is a field with buttons on either side. This allows users to
```

Comment:
*Input group

Reply:
Similar to the comment in #199 this lower-case naming doesn't feel right for named elements. I see why we do it for titles, but for component names this doesn't feel right.

Will update to align with the existing pages though.

Reply:
@vsrixyz @rossmoody passing along these feels for consideration.

Reply:
This has been updated in the new copy.